v2/stacks/feedback/core-ons.yml

```diff
@@ -0,0 +1,51 @@
+services:
+  dp-frontend-feedback-controller:
+    extends:
+      file: ${PATH_MANIFESTS}/core-ons/dp-frontend-feedback-controller.yml
+      service: dp-frontend-feedback-controller
+    depends_on:
+      dp-design-system:
+        condition: service_healthy
+      dp-frontend-router:
+        condition: service_healthy
+  dp-frontend-search-controller:
+    extends:
+      file: ${PATH_MANIFESTS}/core-ons/dp-frontend-search-controller.yml
+      service: dp-frontend-search-controller
+    depends_on:
+      dp-design-system:
+        condition: service_healthy
+      dp-frontend-router:
+        condition: service_healthy
+  dp-feedback-api:
+    extends:
+      file: ${PATH_MANIFESTS}/core-ons/dp-feedback-api.yml
+      service: dp-feedback-api
+    depends_on:
+      mailhog:
+        condition: service_healthy
```

Comment:
I think this file would be better split into:
- supporting services (routers, design-system etc)
- feedback service (api, controller)
- upstream services (search controller)